spec/factories/casa_orgs.rb

```diff
@@ -12,5 +12,15 @@
     trait :with_logo do
       logo { Rack::Test::UploadedFile.new(Rails.root.join("spec", "fixtures", "org_logo.jpeg")) }
     end
+
+    trait :with_placement_types do
+      transient { placement_types { ["Reunification", "Adoption", "Foster Care", "Kinship"] } }
+
+      after(:create) do |org, evaluator|
+        evaluator.placement_types.each do |name|
+          org.placement_types.create!(name: name)
+        end
+      end
```

Comment:
```suggestion
      placements do
        placement_names.map { |name| association(:placement_type, name:) }
      end
```

I _**think**_ this would work, and play well with `build`, `build_stubbed`, or `create`. [see here](https://thoughtbot.github.io/factory_bot/cookbook/has_many-associations.html)

Not a huge deal, but I do think the placement_names change should be made at the least:

```suggestion
      after(:create) do |org, evaluator|
        evaluator.placement_names.each do |name|
          org.placement_types.create!(name: name)
        end
      end
```